lisa/variable.py

```diff
@@ -18,6 +18,8 @@
 _ENV_START = "LISA_"
 _SECRET_ENV_START = "S_LISA_"
 
+_none_type = type(None)
```

Comment:
It's a small perf difference, I guess Python doesn't run `type(None)` only once, if I write it inline.

Reply:
You'll probably find that it will run slower as it will be harder for the compiler to understand the code and then optimize it. At any rate, this seems like a micro-optimization.

Reply:
Agree, it's not possible to be bottleneck in Python. Change to type(None) to improve readability of the code.